tests/plotting/test_plotting.py

```diff
@@ -78,56 +88,10 @@ def using_mesa():
 )
 skip_mesa = pytest.mark.skipif(using_mesa(), reason='Does not display correctly within OSMesa')
 
-# Image regression warning/error thresholds for releases after 9.0.1
-# TODO: once we have a stable release for VTK, remove these.
-HIGH_VARIANCE_TESTS = {
-    'test_add_title',
-    'test_export_gltf',  # image cache created with 9.0.20210612.dev0
-    'test_import_gltf',  # image cache created with 9.0.20210612.dev0
-    'test_opacity_by_array_direct',  # VTK regression 9.0.1 --> 9.1.0
```

Comment:
Did I see correctly that you didn't carry over these comments to the appropriate skips? I think these can be useful for posterity, e.g. when 9.0.1 is no longer supported then we could hypothetically revisit the flakiness of this test. So I'd copy over these comments next to the appropriate per-test skips in the corresponding tests.

Reply:
I didn't, I'll copy them to the appropiate tests.

Reply:
I don't want to sound too nitpicky, but it would be most logical to me to put these comments right before the
```py
verify_image_cache.high_variance_test = True
```
At least if I understand correctly that these comments were explaining the "high variance" part.

Reply:
Not nitpicky at all! I really appreciate your comments. I agree, I was not sure when I moved them but it makes sense to be there.